### DISCUSSION TITLE
Nominate @ForestEckhardt as a contributor

### DISCUSSION THREAD

**Discussion by ForestEckhardt** [paketo-buildpacks/feedback#110] [Dependencies Team]
Hi, I would like to nominate myself as a contributor. I'm a maintainer on the Go, .Net, and Rust teams and would like to have privileges to contribute code directly/re-trigger CI jobs etc. You can see my contributions to the Paketo project at github.com/ForestEckhardt

@paketo-buildpacks/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by sophiewigmore** [comment]
@brayanhenao

**Comment by ryanmoran** [comment]
Congrats @ForestEckhardt! You are now a Dependencies contributor!

**Comment by sophiewigmore** [comment]
Woo!!
